packages/target-tauri/src-tauri/windows/hooks.nsi

```diff
@@ -0,0 +1,42 @@
+!macro URI_SCHEME protocol

```

Comment:
We should probably link to some docs that help understand what this file is about.
Firstly, https://tauri.app/distribute/windows-installer/#extending-the-installer, then maybe an article on "how to register app as default for custom schemes on Windows".